mmdet/apis/ote/apis/detection/configuration.py

```diff
@@ -90,25 +90,38 @@ class __LearningParameters(ParameterGroup):
         )
 
     @attrs
-    class __Postprocessing(ParameterGroup):
-        header = string_attribute("Postprocessing")
+    class __InferenceParameters(ParameterGroup):
+        header = string_attribute("Parameters for inference")
         description = header
 
-        result_based_confidence_threshold = configurable_boolean(
-            default_value=True,
-            header="Result based confidence threshold",
-            description="Confidence threshold is derived from the results",
-            affects_outcome_of=ModelLifecycle.INFERENCE
-        )
-
-        confidence_threshold = configurable_float(
-            default_value=0.35,
-            min_value=0,
-            max_value=1,
-            header="Confidence threshold",
-            description="This threshold only takes effect if the threshold is not set based on the result.",
-            affects_outcome_of=ModelLifecycle.INFERENCE
-        )
+        class_name = selectable(default_value=Models.SSD,
+                                header="Model class for inference",
+                                description="Model classes with defined pre- and postprocessing",
+                                editable=False,
+                                visible_in_ui=True)
+
+        @attrs
+        class __Postprocessing(ParameterGroup):
+            header = string_attribute("Postprocessing")
```

Comment:
As you change the structure of configurable parameters here, please also make corresponding adjustments in [tests](https://github.com/openvinotoolkit/mmdetection/blob/dp/nous_sdk_api/tests/test_ote_api.py#L170-171).